include/swift/Runtime/RuntimeFunctions.def

```diff
@@ -473,6 +473,30 @@ FUNCTION(IsUniquelyReferencedNonObjC_nonNull_bridgeObject,
          ARGS(BridgeObjectPtrTy),
          ATTRS(NoUnwind, ZExt))
 
+// bool swift_isUniquelyReferenced(const void *);
+FUNCTION(IsUniquelyReferenced, swift_isUniquelyReferenced,
+         C_CC, AlwaysAvailable,
```

Comment:
@aschwaighofer made the point that `AlwaysAvailable` is probably wrong here. That also means the code in `GenHeap.cpp` might need some adjustment to check which version to call.

Reply:
I've changed this by adding the necessary availability functions. I've set availability for this feature to Swift 5.6 for now.